code/game/machinery/recycler.dm

```diff
@@ -115,7 +115,7 @@
 		if(istype(AM, /obj/item/organ/brain) || (istype(as_head) && as_head.brain) || (istype(as_mmi) && as_mmi.brain) || istype(AM, /mob/living/brain))
 			emergency_stop(AM)
 		else if(isliving(AM))
-			if((obj_flags & EMAGGED)||!ishuman(AM))
+			if((obj_flags & EMAGGED)||!ishuman(AM)||!isguardian(AM))

```

Comment:
```suggestion
			if((obj_flags & EMAGGED)||!isliving(AM))
```

Safeties should handle all living mobs or just humans (i'd prefer all living mobs)

Just having a guardian stop the recycler seems like an unintuitive mechanic. They're not related to humans in anyway (especially not compared to carbons who also get hurt by this in current commit)

Reply:
can i just remove the || !isliving and just search for the object flags

Reply:
yeah lol duh silly me